admin_app/src/app/login/page.tsx

```diff
@@ -2,7 +2,8 @@
 
 import React, { useState } from "react";
 
-const backendUrl: string = process.env.BACKEND_URL || "http://localhost:8000";
+const backendUrl: string =
+  process.env.NEXT_PUBLIC_BACKEND_URL || "http://localhost:8000";
```

Comment:
Is this used in the login page?

Reply:
Yes! The login page called the backend to authenticate and generate a token